docs/source/conf.py

```diff
@@ -29,6 +29,12 @@
 _release = {}
 exec(compile(open('../../traitlets/_version.py').read(), '../../traitlets/_version.py', 'exec'),_release)
 
+# -- ENSURE: Build docs with current development version of traitlets.
+# REQUIRED-BY: autodoc (autoclass, etc.)
+HERE = os.path.dirname(__file__)
+TOPDIR = os.path.join(HERE, "..", "..")
+sys.path.insert(0, os.path.abspath(TOPDIR))
```

Comment:
No, it doesn't. I got an error regarding the "UseEnum" autoclass. Therefore, I assume that it works with the installed traitlets version instead of using the current development version.

It may work for you, if you run "pip install -e ." (develop mode). If you use the recipe from above, it works on your workspace (even without installing the traitlets package in development mode).